extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -332,7 +335,7 @@ private function getMainPagePartialUrl() {
 	}
 
 	private function isTitleBlacklisted( $title ) {
-		return AccountNavigationController::isBlacklisted( $title );
+		return in_array( $title, static::BLACKLIST_TITLES );
 	}
 
 	private function getReturnToQueryFromQuery( $query ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)